### PR TITLE
Update carbon fee currencies

### PR DESCRIPTION
**Pull request by kimberlybp**
- add fee currencies

### CODE REVIEW

File: cosmos/carbon.json [hunk 1]
```diff
@@ -143,6 +143,50 @@
...
+    },
+    {
+      "coinDenom": "KUJI",
+      "coinMinimalDenom": "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5",
+      "coinDecimals": 6,
+      "coinGeckoId": "kujira",
+      "gasPriceStep": {
+        "low": 0.11538461538461538,
+        "average": 0.14423076923076922,
+        "high": 0.17307692307692307
```
Comment (sarah-thong) [inline cosmos/carbon.json:187]: For this currency, the low step is not high enough, i.e.
(0.11538461538461538*130000)/10^6 ~ 0.0149 KUJI
Screenshot 2023-02-09 at 7 26 22 PM

Probably need to increase low step slightly, e.g. `0.11538461538461539`, and then adjust average (x1.25) and high step (x1.5) accordingly.

File: cosmos/carbon.json [hunk 2]
```diff
@@ -154,6 +198,17 @@
...
+    {
+      "coinDenom": "CANTO",
+      "coinMinimalDenom": "ibc/92E974290AF9E2BC3AEEEC35305C8FD76AC5A22A74CF8D91270FDF5A1C41E861",
+      "coinDecimals": 18,
+      "coinGeckoId": "canto",
+      "gasPriceStep": {
+        "low": 153846153846.15384,
+        "average": 192307692307.6923,
+        "high": 230769230769.23077
+      }
```
Comment (sarah-thong) [inline cosmos/carbon.json:210]: Need to increase low step slightly, e.g. `153846153846.15385`
Refer to comment on line 187

**Comment by HeesungB** [comment]
@kimberlybp @sarah-thong Are you ready to merge?

**Comment by sarah-thong** [comment]
Hi @HeesungB,
Yup, we are ready to merge.

P.S.: just to clarify, the values we are adding here are calculated based on the equation here
https://github.com/chainapsis/keplr-chain-registry/pull/38#issuecomment-1407262739

**Comment by HeesungB** [comment]
@kimberlybp @sarah-thong Thank you for contribution!

**Pull request merged**